doc/mesh_filter/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+install(DIRECTORY launch DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+install(DIRECTORY meshes DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+install(DIRECTORY urdf DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+install(DIRECTORY rviz DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
```

Comment:
According to the current layout all things **except for the launch files** should be installed to `${CATKIN_PACKAGE_SHARE_DESTINATION}/doc/<tutorial>`.
So you don't have to do this weird dance with the additional parameter for the mesh in the xacro.

In theory we could also move all the launch files to this subfolder, but users likely expect them all in one place, so we don't.

Sorry for the confusion. :confused:

Reply:
The tutorial now works for both devel and install workspaces.